pkg/cmd/testdata/out/from-ingress-to-canary-gwapi.all.yaml

```diff
@@ -0,0 +1,62 @@
+Gateways:
```

Comment:
Actually I think this is in more generic since it can be converted into a yaml or json. People can use something like `jq` to parse it. Or maybe we can have an additional flag like `--format` which can make the output in the `kubecl apply` format(Which I think we can add them later). The reason to have a struct to store/nest these resources is that we can filter these GWAPI resources with different types like Gateway, HTTPRoute, etc.

Reply:
I think that having the objects be separate YAML documents within a single file ( that is, separated with `---`) should work just as well though, I think most of the tools will support that, and then it's _also_ directly `kubectl apply`-able.